src/watchdog/observers/inotify_c.py

```diff
@@ -323,7 +323,6 @@ def _recursive_simulate(src_path):
                     path = self._path_for_wd.pop(wd)
                     if self._wd_for_path[path] == wd:
                         del self._wd_for_path[path]
-                    continue
```

Comment:
Is it required to remove that line?

Reply:
Yes, because otherwise the `IN_IGNORE` event isn't added to the `event_list` and I can't handle it in `InotifyBuffer.run`